src/data_hub/filters.py

```diff
@@ -1,65 +1,43 @@
 """module to create filters and conditions to filter rows in databases on the data hub"""
 
-from beartype import beartype
-from deeporigin_data.types.client_list_database_rows_params import (
-    FilterRowFilterNumber,
-    FilterRowFilterText,
-)
-from pydantic import TypeAdapter
-
-
-@beartype
-def numeric_condition(
-    *,
-    column_id: str,
-    value: float | int,
-    operator: str,
-) -> dict:
-    """function to create a valid numeric filter
-
-
-    Args:
-        column_id (str): id of the column
-        value (float | int): value to filter
-        operator (str): operator to use.
-
-    Returns:
-        A dictionary containing a valid condition that can be used to filter database rows
-    """
```

Comment:
this is pointless boilerplate and i don't like depending on the low level SDK for these types